CRM/Activity/BAO/Activity.php

```diff
@@ -2517,7 +2517,12 @@ public static function getContactActivitySelector(&$params) {
           $activity['source_contact_name'] = $values['source_contact_name'];
```

Comment:
This one isn't linked because you just view this contact. So I thought there's also no need for a summary overlay.